integration/conftest.py

```diff
@@ -53,6 +54,11 @@ def pytest_addoption(parser):
         "--coverage", action="store_true", dest="coverage",
         help="Collect coverage statistics",
     )
+    parser.addoption(
+        "--tahoe-version", dest="tahoe_version",
+        help="Tahoe version to install (used for storage-servers and client)",
+        default="tahoe-lafs==1.15.1",
+    )
```

Comment:
I think we have this point to a `requirements/tahoe-integration-1.15.text` file (in the root, #410 will probably put more requirements files there) or something, instead of a version string. (At least for 1.14, I think we might need to bound cryptography/pyopenssl versions; at least I did when testing against that version.) I guess `--tahoe-requirements-file` might thus make a better name?

Reply:
...and then `pip install -r <whatever-was-in-that-option>` right? Yeah if we have to have more stuff in there makes sense

Reply:
Exactly.